examples/react-native-query-suggestions/App.js

```diff
@@ -12,12 +12,14 @@ import {
 } from 'react-native';
 import algoliasearch from 'algoliasearch/lite';
 import {
+  InstantSearch,
+  Configure,
+  Index,
   connectSearchBox,
   connectInfiniteHits,
   connectHits,
   connectRefinementList,
-} from 'react-instantsearch/connectors';
-import { InstantSearch, Configure, Index } from 'react-instantsearch/native';
+} from 'react-instantsearch-native';
```

Comment:
Do you know why it doesn't work as before?

Reply:
Not really, neither do I know why we were using `react-instantsearch/...` instead of `react-instantsearch-native` or `react-instantsearch-dom`.  Was it just outdated?

Reply:
we have https://github.com/algolia/react-instantsearch/blob/master/packages/react-instantsearch/native.js
so it should've worked without this change.

Reply:
True but the other `react-native` test use `react-instantsearch-native` so in either way I think it should be better to make them use the same package. I'll try to see why the other way doesn't work

Reply:
I just realized that this is the only part where this kind of import is used. Has it ever worked outside of this example?

Reply:
The `react-instantsearch-native` import is the most up-to-date way to import the package. It allows better tree shaking IIRC.